galite-core/src/main/kotlin/org/kopi/galite/report/VCCDepthFirstCircuitN.kt

```diff
@@ -15,6 +15,48 @@
  * License along with this library; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  */
+
 package org.kopi.galite.report
 
-abstract class VCCDepthFirstCircuitN 
\ No newline at end of file
+abstract class VCCDepthFirstCircuitN : VCalculateColumn() {
+  /**
+   * Initialisation of the calculation
+   */
+  override fun init() {}
+
+  /**
+   * Evaluates nodes
+   */
+  override fun evalNode(row: VReportRow, column: Int): Any? {
+    return null
+  }
```

Comment:
change to inline method : 
`override fun evalNode(row: VReportRow, column: Int): Any? = null`